src/main/java/com/manardenza/console/ContentsUserMenu.java

```diff
@@ -0,0 +1,261 @@
+package com.manardenza.console;
+
+import com.manardenza.controller.HotelController;
+import com.manardenza.controller.ReservationController;
+import com.manardenza.controller.UserController;
+import com.manardenza.entity.Hotel;
+import com.manardenza.entity.Reservation;
+import com.manardenza.entity.Room;
+import org.slf4j.LoggerFactory;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
+public class ContentsUserMenu {
+
+    public static final String ERROR_INCORRECT_MENU_ITEM_SELECTED = "ERROR: Incorrect menu item selected \n";
+    public static final String ERROR_INVALID_NAME_FORMAT = "ERROR: Invalid name format!\n";
+    private static org.slf4j.Logger log = LoggerFactory.getLogger(ContentsUserMenu.class);
+
+
+    private HotelController hotelController;
+    private ReservationController reservationController;
+    private UserController userController;
+    private VisualUserMenu visualUserMenu;
+
+    public ContentsUserMenu(HotelController hotelController,
+                            ReservationController reservationController, UserController userController,
+                            VisualUserMenu visualUserMenu) {
+        this.hotelController = hotelController;
+        this.reservationController = reservationController;
+        this.userController = userController;
+        this.visualUserMenu = visualUserMenu;
+    }
+
+    public void mainMenu() {
+        VisualUserMenu.printListInConsole(ListMenu.getSiteHeader(), null);
+        VisualUserMenu.printListInConsole(ListMenu.getAuthorizationHeader(), ListMenu.getAuthorizationMenu());
+        Integer action;
+        do {
+            action = Integer.parseInt(visualUserMenu.getValidInputFromUser("Choose action: ", InputType.INTEGER));
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Choose action: " 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)